src/app/tests/suites/OperationalCredentialsCluster.yaml

```diff
@@ -23,10 +23,15 @@ tests:
       command: "readAttribute"
       attribute: "SupportedFabrics"
       response:
-          value: 16
+          constraints:
+              type: uint8
+              minValue: 5
```

Comment:
Let's use 4 for now, since some platforms use 4 and the minimum of 5 (or any, really) isn't in spec yet.